src/components/ContributionsEpic.tsx

```diff
@@ -88,10 +100,19 @@ export type EpicMetadata = {
     referrerUrl: string;
 };
 
+export type EpicTargeting = {
```

Comment:
This PR doesn't actually use targeting yet, so should we remove it for now? I.e. generally it is better to introduce code when it is used so that a) it is easier to review (as clearer how it meets requirements) and b) avoids risk of YAGNI.

Reply:
We are making use of the `EpicTargeting` type in this PR (in server.tsx) to build a targeting object to be passed downstream - but we're not implementing any targeting logic here, so not 100% on either side of the fence. I would like to keep it on this PR otherwise we'd have to remove the code that handles that data off the POST object but I know this isn't "releasable" as it currently stands.

I do agree though that the type declaration should be moved out of this file. I'm playing around with an Epic wrapper component where I thought the targeting logic would go - perhaps this can be moved there in a subsequent PR (once we have that wrapper in place)?